radio/src/gui/128x64/view_main.cpp

```diff
@@ -297,6 +297,36 @@ void onMainViewMenu(const char *result)
   }
 }
 
+void displaySwitch(coord_t x, coord_t y, int width, unsigned int index)
```

Comment:
It looks strange to me to have 2 different functions `drawSwitch` and `displaySwitch`